pkg/webhook/pod/v1/pod_default.go

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
It seems that `mutate webhook` is not needed for nodepool autonomy, so how about remove this webhook for improving webhook efficiency?